### ISSUE TITLE
worktree.Status not honouring .gitignore

### ISSUE THREAD

**Issue by jaxxstorm** [go-git/go-git#91]
I have a function that checks the working tree's current status:

```
status, err := worktree.Status()
// more stuff
```

in a particular git repo, I have a `node_modules` directory in a sub directory:

```
sdk/nodejs/node_modules
```

I added the following to my `.gitignore`

```
**/node_modules
```

Then I ran `status.IsClean()`

However, for whatever reason, the `.gitignore is not being honoured for 2 files:

```
?? sdk/nodejs/node_modules/yn/index.js
?? sdk/nodejs/node_modules/yn/package.json
```

No matter what I do (including explicitly adding `sdk/nodejs/node_modules/yn/index.js` to the .gitignore` the working tree comes back dirty. Any thoughts?

**Comment by ZerGo0** [comment]
Same here, I did some debugging and it seems like for whatever reason it breaks .gitignore if there is more than 1 line?

**Comment by AriehSchneier** [comment]
@pjbgf looks like this was fixed in #115
